etc/adminhtml/system.xml

```diff
@@ -46,6 +46,11 @@
                     <comment>This field has to match the one set in dashboard.razorpay.com/webhooks</comment>
                     <config_path>payment/razorpay/webhook_secret</config_path>
                 </field>
+                <field id="webhook_wait_time" translate="label" type="text" sortOrder="6" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
@ramth05 will this consider by default 300 seconds?

Reply:
yes